src/app/auth/components/LogIn/LogIn.tsx

```diff
@@ -172,12 +180,14 @@ export default function LogIn(): JSX.Element {
 
       if (castedError.message.includes('not activated') && auth.isValidEmail(email)) {
         navigationService.history.push(`/activate/${email}`);
-      } else {
-        // analyticsService.signInAttempted(email, castedError);
       }
 
       setLoginError([castedError.message]);
       setShowErrors(true);
+      if (castedError.message.includes('Your account has been blocked for security reasons. Please reach out to us')) {
+        await sendUnblockAccountEmail(email);
+        navigationService.push(AppView.BlockedAccount, { email: email });
```

Comment:
I have seen that the view `BlockedAcccount` requires a token, where do you get it from in this case if the navigation is direct after making the unlock request?

Reply:
It is optional. 

1. Exists: show a loader and send the token to the backend to unlock the account linked to it.  If the request is successfull then go to /login, otherwise show expired view. The backend will get all the needed info from the jwt so the front is agnostic to this.
2. No token: show the view and let the user to resend the email after 30 seconds.